### DISCUSSION TITLE
Get user_metadata from Auth0

### DISCUSSION THREAD

**Discussion by dmarsola-irdeto** [nextauthjs/next-auth#5519] [Help]
For the auth0 provider, would it be possible to retrieve and make changes to the user_metadata information?

**Comment by oskarrough** [comment]
The `user_metadata` is not available by default when using Auth0 in general. They recommend you make an "Action" script in the dashboard, which adds the meta data the the `user` on login. See https://auth0.com/docs/manage-users/user-accounts/metadata/manage-user-metadata.

However, with "auth0.js" the `session.user` only includes `{name, email, image}`. I do not know where to adjust this.
